sections/designRules.md

```diff
@@ -617,49 +603,237 @@ Changes in APIs are inevitable. APIs should therefore always be versioned, facil
          <p>The version number MUST be returned in an HTTP response header named <code>API-Version</code> (case-insensitive) and SHOULD not be prefixed.</p>
          <div class="example">
             <p>An example of an API version response header:</p>
-            <pre>API-Version: 1.0.2</pre>
+            <pre class="nohighlight">API-Version: 1.0.2</pre>
          </div>
       </dd>
-      <dt>Implications</dt>
-      <dd>
-         This rule can be tested automatically and an example of the test is included in the automatic tests on <a href="https://developer.overheid.nl/">developer.overheid.nl</a>. The specific tests are published in the [[ADR-Validator]] repository.
-      </dd>
       <dt>How to test</dt>
       <dd>
-         A response MUST include the header "API-Version".
+         A response includes a header "API-Version" with a number matching the version number set in the `info.version` field of the OpenAPI Description.
       </dd>
    </dl>
 </div>
 
 ## Transport Security
 
-Transport security is essential to safeguard the confidentiality, integrity, and authenticity of data during its transmission.
+This section describes security principles, concepts and technologies to apply when working with APIs.
+Controls need to be applied for the security objectives of integrity, confidentiality and availability of the API (which includes the services and data provided thereby).
+The [architecture section of the API strategy](https://docs.geostandaarden.nl/api/API-Strategie-architectuur/) contains architecture patterns for implementing Transport security.
+
+The scope of this section is limited to generic security controls that directly influence the visible parts of an API.
+Effectively, only security standards directly applicable to interactions are discussed here.
+
+In order to meet the complete security objectives, every implementer MUST also apply a range of controls not mentioned in this section.
 
-<div class="rule" id="/core/transport-security" data-type="technical">
-  <p class="rulelab">Apply the transport security module</p>
+Note: security controls for signing and encrypting of application level messages are part of separate extensions: [Signing](https://geonovum.github.io/KP-APIs/API-strategie-modules/signing-jades/) and [Encryption](https://geonovum.github.io/KP-APIs/API-strategie-modules/encryption/).
+
+<span id="api-11"></span>
+<div class="rule" id="/core/transport/tls" data-type="technical">
+  <p class="rulelab">Secure connections using TLS</p>
   <dl>
     <dt>Statement</dt>
     <dd>
-      The [[[ADR-TS]]] version 1.0.x MUST be applied.
+      <p>One should secure all APIs assuming they can be accessed from any location on the internet. Information MUST be exchanged over TLS-based secured connections. No exceptions, so everywhere and always. This is <a href="https://wetten.overheid.nl/BWBR0048156/2023-07-01">required by law</a>.
+      <p>One MUST follow the latest NCSC guidelines [[NCSC 2021]].
```

Comment:
Recent is er een nieuwe versie gepubliceerd:
https://www.ncsc.nl/documenten/publicaties/2025/juni/01/ict-beveiligingsrichtlijnen-voor-transport-layer-security-2025-05
Ref aanpassen naar correcte "latest"?

Reply:
Dit moet eerst in `develop` alvorens we kunnen patchen.